youtube_dl/extractor/seeso.py

```diff
@@ -0,0 +1,91 @@
+    _VALID_URL = r'https?://(?:www\.)?seeso\.com/view/episode/(?P<id>[0-9]+)'
+    _TEST = {
+        'params': {
+            'username': 'emailhere',
+            'password': 'passwordhere'
+        },
+        'url': 'https://www.seeso.com/view/episode/799241283849',
+        'info_dict': {
+            'id': '799241283849',
+            'ext': 'mp4',
+            'series': 'Bajillion Dollar Propertie$',
+            'title': 'Farsi Lessons',
+            'description': ("Amir leads Victoria into a trap, the Bros meet a super obnoxious bro, "
+                            "and rival brokers Serge and Gio intimidate Glenn."),
+            'thumbnail': 'https://chaosic.akamaized.net/NBCOTT_-_Production/360/907/160831_3092487_Farsi_Lessons.jpg',
+        }
+    }
+
+    def _real_extract(self, url):
+        username, password = self._get_login_info()
+        video_id = self._match_id(url)
+
+        if username is None or password is None:
+            return
+
+        _API_AUTH_URL = "https://www.seeso.com/api/v3/auth/login"
+        auth_json = {
+            "username": username,
+            "password": password,
+            "userOnly": True,
+            "isRememberMe": True,
+            "isWeb": True
+        }
+
+        # Send auth POST request and get token from response
+        auth_request = sanitized_Request(_API_AUTH_URL, urlencode_postdata(auth_json))
+        auth_response = json.loads(self._download_webpage(auth_request, '', note='Getting auth token...'))
+        auth_token = auth_response.get('user').get('token')
+
+        # Use the public unauthenticated API to get the video's info
+        _VIDEO_INFO_URL = 'https://feed.theplatform.com/f/NZILfC/nbcott-prod-all-media?byAvailabilityState=' \
+                          'available&byId=%s&form=cjson'
+        request = sanitized_Request(_VIDEO_INFO_URL % video_id)
+        json_data = self._download_json(request, video_id)
+        entry = json_data.get('entries')[0]
+
+        # Template fields
+        series = entry["nbc-chaos$show"]                    # Show name
+        title = entry["nbc-chaos$shortTitle"]               # Episode Name
+        season_number = entry["nbc-chaos$seasonNumber"]
+        episode_number = entry["nbc-chaos$episodeNumber"]
+        thumbnail = entry["defaultThumbnailUrl"]
+        description = entry["nbc-chaos$shortDescription"]
+
+        # Got the show's public URL. Now we need to parse out the videoID
+        public_url_id = os.path.split(urlparse.urlparse(entry["publicUrl"]).path)[-1]
+
+        # Get the master m3u8 which lists formats
+        m3u8_url = 'https://link.theplatform.com/s/NZILfC/media/{0}?feed=All%20Media%20Feed&auth={1}' \
+                   '&vpaid=script,flash&formats=m3u,mpeg4'.format(public_url_id, auth_token)
+        formats = []
+        for entry in self._extract_m3u8_formats(m3u8_url, video_id, m3u8_id='m3u8', ext='mp4'):
+            formats.append(entry)
+
+        self._sort_formats(formats)
```

Comment:
`CBSIE` does not use `ThePlatformFeedIE._extract_feed_info` now.
it was used before like this:
https://github.com/rg3/youtube-dl/blob/cdfc187cd5bd163e7e67ca0c02108380cc06c180/youtube_dl/extractor/cbs.py#L21-L31
the code for seeso should look similar to(didn't test it but it should work):
```python
return self._extract_feed_info(
    'NZILfC', 'nbcott-prod-all-media', 'byId=' + video_id, lambda entry: {
        'series': entry.get('nbc-chaos$show'),
        'season_number': int_or_none(entry.get('nbc-chaos$shortTitle')),
        'episode': entry.get('nbc-chaos$shortTitle'),
        'episode_number': int_or_none(entry.get('nbc-chaos$episodeNumber')),
    }, {
        'Video': {
            'auth': auth_token,
        }
    })
```